eclair-core/src/main/scala/fr/acinq/eclair/channel/fund/InteractiveTxBuilder.scala

```diff
@@ -201,6 +201,7 @@ object InteractiveTxBuilder {
             params: Params,
             commitTxFeerate: FeeratePerKw,
             remoteFirstPerCommitmentPoint: PublicKey,
+            remoteSecondPerCommitmentPoint: PublicKey,
```

Comment:
Providing a `remoteNextCommitInfo` would save a `get` by the caller, but it's less pure because we make the implicit assumption that there are no pending remote commit. I'll have the same kind of stuff on the splices PR.